packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -99,6 +101,13 @@ export default class CSSEditor extends Foundation<
         return pick(this.props.data, ["color"]);
     }
 
+    /**
+     * Gets the border value
+     */
+    private getBorderData(): CSSBorderValues {
+        return pick(this.props.data, ["border"]);
```

Comment:
this should not be "border", should be "borderWidth", "borderColor", "borderStyle"